sdk/messaging/azeventhubs/internal/namespace.go

```diff
@@ -457,7 +456,7 @@ func (ns *Namespace) GetEntityAudience(entityPath string) string {
 func (ns *Namespace) getUserAgent() string {
 	userAgent := rootUserAgent
 	if ns.userAgent != "" {
```

Comment:
So `ns.userAgent` is the user-provided application ID?

Reply:
Yeah, I never changed the internal field name but it maps from this exposed option: 
* (option) https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/messaging/azeventhubs/consumer_client.go#L46
* (copying) https://github.com/Azure/azure-sdk-for-go/blob/1ad22a964ea16c159e7e3f28263c940667d61519/sdk/messaging/azeventhubs/consumer_client.go#L249